app/assets/stylesheets/partials/_gdsBase.scss

```diff
@@ -9,8 +9,6 @@ needs to be behind the dropdown arrow.
 .govuk-template__body {
   margin: 0;
   background-color: #fff;
-  position: relative;
```

Comment:
so with position absolute this works? Interesting

Reply:
Nope, There is another change required, which I changed locally in the browser console,

Reply:
Now PR contains this, but this require testing

Reply:
You might as well remove the `.govuk-template__body`, as that was only there to override the `z-index`.

Reply:
This works...